toolkit/tools/scheduler/schedulerutils/buildlist.go

```diff
@@ -67,8 +67,8 @@ func ReadReservedFilesList(path string) (reservedFiles []string, err error) {
 
 	file, err := os.Open(path)
 	if err != nil {
-		logger.Log.Errorf("Failed to open file manifest %s with error %s", path, err)
-		return nil, err
+		reservedFiles, err = nil, fmt.Errorf("failed to open file manifest (%s):\n%w", path, err)
```

Comment:
nit: `err = fmt.Errorf("failed to open file manifest (%s):\n%w", path, err)`

`reservedFiles` hasn't been assigned to yet. So, it will already have a value of `nil`.

Reply:
Oh, thanks for pointing that out, I was not aware of it.
I'll keep it still. I would prefer the code to lay out variable values clearly, than leave the reader guessing. Saves time in debugging/ future dev